src/offliner.js

```diff
@@ -264,6 +267,21 @@
   };
 
   /**
+   * Check if there is an activation pending. If so, offliner dispatches an
+   * activation pending request.
+   *
+   * @method _checkForActivationPending
+   * @private
+   */
+  Offliner.prototype._checkForActivationPending = function () {
```

Comment:
Nit: Instead of defining another function, you could directly use `_activate` (which does the same thing).